### ISSUE TITLE
The results of attn in the prediction outputs

### ISSUE THREAD

**Issue by daxiang415** [QData/spacetimeformer#49]
Dear authors

I recently make the application of spacetimeformer, but it seems a little bit confused for me to understand the attn in the outputs. 

Would you mind give me an introduction of how to utilize the logits, labes and attn in the model outputs.

**Comment by daxiang415** [comment]
Sorry, I just figured out by myself, thank you.
